components/data-table/private/head.jsx

```diff
@@ -56,10 +56,10 @@ const DataTableHead = createReactClass({
 	render () {
 		return (
 			<thead>
-				<tr className="slds-text-title--caps">
+				<tr className="slds-line-height_reset">
```

Comment:
What's the reason for moving `slds-text-title_caps` down on level?  The SLDS sample has it on the `<tr>`, and `slds-text-title_caps` includes `line-height: 1.25` so we wouldn't need to have `slds-line-height_reset` here?

Reply:
I'm following the spec of the advanced variant. I'm seeing `slds-text-title--caps` on the `th` and `slds-line-height_reset` on the `tr`.

![screen shot 2018-05-22 at 3 12 03 pm](https://user-images.githubusercontent.com/1290832/40390463-8fde2264-5dd2-11e8-9cde-650ae03e54a5.png)